phpstan.neon

```diff
@@ -23,10 +22,14 @@ parameters:
         - '#Variable \$config in isset\(\) is never defined#'
         - '#Call to static method id\(\) on an unknown class PHPUnit_Runner_Version#'
         - '#Call to an undefined method DateTimeInterface::i18nFormat\(\)#'
-        - '#Call to an undefined method object::__toString\(\)#'
-        - '#Call to an undefined method object::toArray\(\)#'
-        - '#Call to an undefined method object::__debugInfo\(\)#'
         - '#Cannot call method lastInsertId\(\) on null#'
+        - '#Call to an undefined method Cake\\Auth\\Storage\\StorageInterface::getConfig\(\)#'
+        - '#Call to an undefined method Cake\\Auth\\Storage\\StorageInterface::setConfig\(\)#'
+        - '#Variable \$_SESSION in isset\(\) always exists and is not nullable#'
+        - '#Call to an undefined method Traversable::getInnerIterator\(\)#'
+        - '#PHPDoc tag @throws with type PHPUnit\\Exception is not subtype of Throwable#'
+        - '#Call to an undefined method Cake\\Console\\ConsoleOutput\|PHPUnit_Framework_MockObject_MockObject::messages\(\)#'
+        - '#Binary operation "\+=" between \(array\&hasOffset\(.*\)\) and array results in an error#'
```

Comment:
This can be removed after next release `0.10.3` of phpstan. It's a bug in phpstan phpstan/phpstan#1302.